File: Demo-LLP-MIP/Demo-LLP-MIP-PHP/v1/ipay/argument/supportbank/SupportBankQueryParams.php
<?php


class SupportBankQueryParams
{
    public $mch_id;
    public $pay_type;
    public $bank_code;
    public $card_type;
    public $biz_scene;

    public function __construct()
    {
        $this->mch_id = null;
        $this->pay_type = null;
        $this->bank_code = null;
        $this->card_type = null;
        $this->biz_scene = null;
    }

    public function to_dict() {
        $data = [];
        foreach ($this as $key => $value) {
            if ($value !== null) {
                $data[$key] = $value;
            }
        }
        return json_encode($data);
    }

}

File: Demo-LLP-MIP/Demo-LLP-MIP-Python/v1/ipay/argument/supportbank/SupportBankQueryParams.py
class SupportBankQueryParams:
    def __init__(self):
        self.mch_id = None
        self.pay_type = None
        self.bank_code = None
        self.card_type = None
        self.biz_scene = None

    def to_dict(self):
        return {
            "mch_id": self.mch_id,
            "pay_type": self.pay_type,
            "bank_code": self.bank_code,
            "card_type": self.card_type,
            "biz_scene": self.biz_scene
        }